src/lesson03/Task02.java

```diff
@@ -0,0 +1,41 @@
+package lesson03;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Task02 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Input Data:");
+        System.out.print("number=");
```

Comment:
в такому виді краще вже обєднати 
```suggestion
        System.out.print("Input Data:\nnumber=");
```